Formula/qtkeychain.rb

```diff
@@ -47,7 +46,8 @@ def install
       [
         "-fPIC",
         "-L#{Formula["qt@5"].opt_lib}", "-lQt5Core",
-        "-Wl,-rpath,#{Formula["qt@5"].opt_lib}"
+        "-Wl,-rpath,#{Formula["qt@5"].opt_lib}",
```

Comment:
`qt@5`, `Qt5Core`, `qt5keychain`, etc. references will need to be updated.

Also will need a revision bump due to dependency change.

Reply:
Whoops this did need quite a few other changes.  I'm confirming the proper names for everything now and will then push the necessary changes.